### PR TITLE
allow symfony 7

### PR DESCRIPTION
**Pull request by dbu**
please do not merge before symfony 7 is released

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -46,6 +46,10 @@ jobs:
             phpunit-version: 9
             symfony-version: '^6'
             stability: 'dev'
+          - php-version: '8.3'
+            phpunit-version: 9
+            symfony-version: '^7@dev'
```
Comment (dbu) [inline .github/workflows/ci.yml:51]: once symfony 7 is released, remove the `@dev` before merging.
```suggestion
            symfony-version: '^7'
```

Comment (jeanmonod) [reply .github/workflows/ci.yml:51]: What is the expected release date?

Comment (dbu) [reply .github/workflows/ci.yml:51]: november 2023.

i wanted to check with a project what the upcoming release will do, and had the conflict with the rmt so i thought i just check that as well and use that branch for the other project...

**Comment by dbu** [comment]
@jeanmonod symfony 7 has been released. good thing we waited, there was one more method that got a return type declaration in Symfony 7. and not repeating the return type is a fatal error with PHP 8.3

**Comment by rusted-love** [comment]
What is blocking current PR?

**Comment by rusted-love** [comment]
@jeanmonod @leorott Can somebody merge this?

**Comment by jeanmonod** [comment]
Hi all,
It's merge and released under 1.7.3
Sorry for the delay
And thanks @dbu for the contribution here! 👍

**Pull request merged**